lib/docker2aci.go

```diff
@@ -53,11 +53,13 @@ type Docker2ACIBackend interface {
 // If the squash flag is true, it squashes all the layers in one file and
 // places this file in outputDir; if it is false, it places every layer in its
 // own ACI in outputDir.
+// It will use the temporary directory specified by tmpDir, or the default
+// temporary directory if tmpDir is "".
 // username and password can be passed if the image needs authentication.
 // It returns the list of generated ACI paths.
-func Convert(dockerURL string, squash bool, outputDir string, username string, password string) ([]string, error) {
+func Convert(dockerURL string, squash bool, outputDir string, tmpDir string, username string, password string) ([]string, error) {
```

Comment:
Function with six parameters, ugh. This is getting out of hand.

Feel free to ignore this comment, of course.